### ISSUE TITLE
add Corollary 2.5 from Gewand

### ISSUE THREAD

**Issue by Almanzoris** [pi-base/data#682]
![image](https://github.com/pi-base/data/assets/149833592/13b51391-f684-4c29-8a2e-e70f541901b8)

Munkres, J. R. (2013). Topology: Pearson New International Edition.

**Comment by prabau** [comment]
For comparison, some spaces with these properties already in pi-base:
https://topology.pi-base.org/spaces?q=t3%2B%7EFunctionally+Hausdorff

Not determined yet (note the use of the question mark in the search):
https://topology.pi-base.org/spaces?q=t3%2B%3FFunctionally+Hausdorff
That's S92 from issue #681, and S87 (deleted Dieudonne plank), which should be Tychonoff, as a subspace of Dieudonne plank ...

**Comment by Almanzoris** [comment]
Yeah, S87 is P6.

**Comment by Almanzoris** [comment]
I am also reading about Brian's example (S171) (https://mathoverflow.net/a/416752/506958) and noticed that it could be added that it is not regular, since [first countable + T_3 + Lindelöf + scattered --> countable] and this space is [first countable + Lindelöf + scattered + uncountable].

Furthermore, from how he deduces that the space is T_2 (the rational Euclidean-open intervals are still open in the topology, i.e., it refines the usual topology on R), we could also deduce that the space is Functionally Hausdorff.

**Comment by Almanzoris** [comment]
Actually, as ~regular is automatically deduced from that, but it didn't show up in the list, that means that the theorem could be added, instead of just setting regular to false in the space.

**Comment by prabau** [comment]
> Actually, as ~regular is automatically deduced from that, but it didn't show up in the list, that means that the theorem could be added, instead of just setting regular to false in the space.

I was thinking about the same thing.  I agree that it's better to deduce that Brian's example is not regular from a theorem.  https://mathoverflow.net/questions/416331 claims that [first countable + T_3 + Lindelöf + scattered --> countable], which pi-base cannot deduce at the moment.  Why is that true?  Can some of the hypotheses be weakened?  Maybe mathse has a related question already?

**Comment by Almanzoris** [comment]
Already, since scattered implies T_0, T_3 can be relaxed to regular.

**Comment by Almanzoris** [comment]
Check:
M. E. Gewand, “The Lindelöf degree of scattered spaces and their products,” Journal of the Australian Mathematical Society (Series A) 37 (1984), 98–105.

**Comment by prabau** [comment]
Very nice!  I have not read the article yet, but it would be good to have a more direct proof of this Corollary 2.5.  Worth asking on mathse.

In any case, the property "Every point is a $G_\delta$" (same as "countable pseudocharacter") in an interesting property that should be added to pi-base.  It implies T1 (I think), and is implied by T1 + first countable.  Engelking should have various results about it.

You have now opened two good issues with many suggestions.  Is that something you could work on, even if part of it?  If so, what is your plan for implementation?

**Comment by prabau** [comment]
> In our case, since (X, T) is regular, each point has a closed neighborhoods basis, and as it is first countable, then each point has a countable closed neighborhoods basis. Now, as scattered implies T_0 and (X, T) is regular, (X, T) is T_3 so, it is also T_2 and, therefore, each singleton is equal to the intersection of its closed neighborhoods. So each singleton is a G_{\delta} set, as it is equal to the intersection of the closed neighborhoods of a countable closed neighborhoods basis that the point has.

Not sure I completely follow this.  Isn't easier to use the fact that in a T1 first countable space each point is a $G_\delta$?

**Comment by Almanzoris** [comment]
> > In our case, since (X, T) is regular, each point has a closed neighborhoods basis, and as it is first countable, then each point has a countable closed neighborhoods basis. Now, as scattered implies T_0 and (X, T) is regular, (X, T) is T_3 so, it is also T_2 and, therefore, each singleton is equal to the intersection of its closed neighborhoods. So each singleton is a G_{\delta} set, as it is equal to the intersection of the closed neighborhoods of a countable closed neighborhoods basis that the point has.
> 
> Not sure I completely follow this. Isn't easier to use the fact that in a T1 first countable space each point is a Gδ?

Yeah, you are right.

**Comment by Almanzoris** [comment]
> Very nice! I have not read the article yet, but it would be good to have a more direct proof of this Corollary 2.5. Worth asking on mathse.
> 
> In any case, the property "Every point is a Gδ" (same as "countable pseudocharacter") in an interesting property that should be added to pi-base. It implies T1 (I think), and is implied by T1 + first countable. Engelking should have various results about it.
> 
> You have now opened two good issues with many suggestions. Is that something you could work on, even if part of it? If so, what is your plan for implementation?

Sure, I will try to work on it or help to. I am unsure yet how should we proceed with this. But when it comes to S92 and S87, we can make the respective updates directly, and, regards Brian example, the property Functionally Hausdorff can be updated too, though there are more examples that are a refinement of the usual topology, so maybe this could be made as a property too and automatize the consequent implications it makes.

**Comment by prabau** [comment]
A general comment about implementing things.  A big bang approach is to be avoided.  It is much more manageable to have several smaller independent pull requests (PR) that can be reviewed and incorporated more quickly.

So reasonable to me would be one PR for S87, which can also update the related S191 (Dieudonne plank). (in particular, S191 is Tychonoff, hence so is S87 since it's a hereditary property.)  Another PR for S92.  (Note: these can be done in parallel on branches forked off the same main branch, as they are independent of each other.)  I would leave the Munkres example alone for now, but can be done later if desired.

For Brian's example, there is no rush.  I would first do a separate PR to introduce the countable pseudocharacter property and a few basic theorems.  Once this is released, another PR for the Corollary 2.5 you mentioned.  And finally another PR for changes to Brian's example.  (And also later more PR for cleanup/tightening of traits of existing spaces based on the new pseudocharacter property).

These are just suggestions.

There is no rush for any of that, so you can do it as slow or as fast as you want.  I try to give comments in a timely manner. Unfortunately, it seems that some of the other reviewers are often busy and it can take time to get feedback :-(

**Comment by Almanzoris** [comment]
Alright, I will follow the plan at my pace. Thank you for your help and labour. As a noob, it is hard for me to work xD.

By experience, your feedback is always great enough, so it is alright, although I know that it is always good to have more feedback when it is possible.

**Comment by StevenClontz** [comment]
Just chiming in to say "points-$G_\delta$" is a good property to have in pi-Base; I recall it coming up in the study of various limited info strategies for the Rothberger covering game I did a few years back. And of course $T_1$+[$G_\delta$ space](https://topology.pi-base.org/properties/P000132) implies points-$G_\delta$.

**Comment by prabau** [comment]
Added #692.  It could possibly be useful to get a direct proof of Gewand's Corollary 2.5.

**Comment by StevenClontz** [comment]
The conjecture is [scattered+points Gdelta => locally countable](https://topology.pi-base.org/spaces?q=%24G_%5Cdelta%24+space%2BT_1%2Bscattered%2B%7ELocally+countable)?

**Comment by StevenClontz** [comment]
> Check: M. E. Gewand, “The Lindelöf degree of scattered spaces and their products,” Journal of the Australian Mathematical Society (Series A) 37 (1984), 98–105.
> 
> ![image](https://private-user-images.githubusercontent.com/149833592/346728091-3596ba73-03e3-4ba3-8dc2-6e28b782b862.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjA5NjQzNDAsIm5iZiI6MTcyMDk2NDA0MCwicGF0aCI6Ii8xNDk4MzM1OTIvMzQ2NzI4MDkxLTM1OTZiYTczLTAzZTMtNGJhMy04ZGMyLTZlMjhiNzgyYjg2Mi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNzE0JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDcxNFQxMzM0MDBaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT03MjNjZjhhOTA1YmVkMjUxZjdiODc3YTVhYjY5ZDc1Y2QzZDZiMjQ1Nzk1NTExYmE4NWQ4ZTc0MGY5Mzk5ZWY4JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.rjxhf_L6auSQdtlWLu94vLi1rY0qx2kl095D232UkL0)


Is the assertion that $X$ is finite (not just countable) a typo? Wouldn't a converging sequence be (countably) infinite, scattered, Lindelöf (in fact compact), with each point a $G_\delta$.

**Comment by StevenClontz** [comment]
I asked this quesiton on Math SE: https://math.stackexchange.com/questions/4945902/must-scattered-spaces-with-points-g-delta-be-locally-countable

**Comment by prabau** [comment]
> > Check: M. E. Gewand, “The Lindelöf degree of scattered spaces and their products,” Journal of the Australian Mathematical Society (Series A) 37 (1984), 98–105.
> > ![image](https://private-user-images.githubusercontent.com/149833592/346728091-3596ba73-03e3-4ba3-8dc2-6e28b782b862.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjA5NjQzNDAsIm5iZiI6MTcyMDk2NDA0MCwicGF0aCI6Ii8xNDk4MzM1OTIvMzQ2NzI4MDkxLTM1OTZiYTczLTAzZTMtNGJhMy04ZGMyLTZlMjhiNzgyYjg2Mi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNzE0JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDcxNFQxMzM0MDBaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT03MjNjZjhhOTA1YmVkMjUxZjdiODc3YTVhYjY5ZDc1Y2QzZDZiMjQ1Nzk1NTExYmE4NWQ4ZTc0MGY5Mzk5ZWY4JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.rjxhf_L6auSQdtlWLu94vLi1rY0qx2kl095D232UkL0)
> 
> Is the assertion that X is finite (not just countable) a typo? Wouldn't a converging sequence be (countably) infinite, scattered, Lindelöf (in fact compact), with each point a Gδ?

Yeah, that's a typo.  The result is $|X|\le\omega$.

**Comment by Almanzoris** [comment]
Should we ask further in mathse for a more direct proof of the Corollary 2.5?

**Comment by prabau** [comment]
I'll take a closer look later today.

**Comment by prabau** [comment]
I took a look and I think it would be worth asking for a more direct proof of Cor 2.5 on mathse.  That will simplify the argument a little.

**Comment by StevenClontz** [comment]
I think the original topic here is addressed, so I'm closing this issue. Please open a new issue to continue any of the side discussions though!

**Comment by prabau** [comment]
Hmm, the main thing here was going to add Corollary 2.5 from Gewand.  Seem premature to close this.

**Comment by StevenClontz** [comment]
No worries, reopening with appropriate title for the issue.

**Comment by Almanzoris** [comment]
@prabau I have read that part of the book again.

Let $(X, \top)$ be a $T_3$ (regular is enough) scattered Lindelöf space with countable pseudocharacter.

(Before going on, $X_{\delta}$ denotes the topological space of $X$ with the topology generated by its $G_{\delta}$ sets).

As a consequence of the Theorem 2.2 (X is $T_3$, scattered and Lindelöf), $X_{\delta}$ is Lindelöf. Now, as each point of $X$ is a $G_{\delta}$ set, then we can consider the cover of $X_{\delta}$ $\mathcal{A} = X$, as $X_{\delta}$ is Lindelöf, $X$ needs to be countable.

**Comment by Almanzoris** [comment]
This following thing is off-topic:

I understand the symbol $\subset$ as $\subsetneq$. But, I know that some authors make use of $\subset$ as $\subseteq$.

What notation is more accepted or extended?

**Comment by prabau** [comment]
> I understand the symbol ⊂ as ⊊. But, I know that some authors make use of ⊂ as ⊆.
> 
> What notation is more accepted or extended?

That's why I usually use $\subseteq$ to mean inclusion.  There is no ambiguity when using that notation.

**Comment by prabau** [comment]
> @prabau @StevenClontz I have read that part of the book again.
> 
> Let (X,τ) be a T3 (regular is enough) scattered Lindelöf space with countable pseudocharacter.
> 
> (Before going on, Xδ denotes the topological space of X with the topology generated by its Gδ sets under the topology τ).
> 
> As a consequence of the Theorem 2.2 (X is T3, scattered and Lindelöf), Xδ is Lindelöf. Now, as each point of X is a Gδ set, then Xδ is discrete and we can consider the cover of Xδ: A={{x}:x∈X}. Finally, as Xδ is Lindelöf, X needs to be countable.

Just ask a question on mathse, so we can get a self-contained proof of the corollary, without having to rely on the other results of the article.  I can answer with a direct proof of it that is not difficult.  Or what do you prefer to do?

**Comment by Almanzoris** [comment]
> > I understand the symbol ⊂ as ⊊. But, I know that some authors make use of ⊂ as ⊆.
> > What notation is more accepted or extended?
> 
> That's why I usually use ⊆ to mean inclusion. There is no ambiguity when using that notation.

Yeah, I asked this after reading the definition of the space S14.

**Comment by Almanzoris** [comment]
> > @prabau @StevenClontz I have read that part of the book again.
> > Let (X,τ) be a T3 (regular is enough) scattered Lindelöf space with countable pseudocharacter.
> > (Before going on, Xδ denotes the topological space of X with the topology generated by its Gδ sets under the topology τ).
> > As a consequence of the Theorem 2.2 (X is T3, scattered and Lindelöf), Xδ is Lindelöf. Now, as each point of X is a Gδ set, then Xδ is discrete and we can consider the cover of Xδ: A={{x}:x∈X}. Finally, as Xδ is Lindelöf, X needs to be countable.
> 
> Just ask a question on mathse, so we can get a self-contained proof of the corollary, without having to rely on the other results of the article. I can answer with a direct proof of it that is not difficult. Or what do you prefer to do?

Sure, I will ask it.

**Comment by prabau** [comment]
> > That's why I usually use ⊆ to mean inclusion. There is no ambiguity when using that notation.
> 
> Yeah, I asked this after reading the definition of the space S14.

No big deal.  We can fix this together with some other stuff at any time.
